frappe/database.py

```diff
@@ -50,7 +50,7 @@ def connect(self):
 		"""Connects to a database as set in `site_config.json`."""
 		warnings.filterwarnings('ignore', category=MySQLdb.Warning)
 		self._conn = MySQLdb.connect(user=self.user, host=self.host, passwd=self.password,
-			use_unicode=True, charset='utf8mb4')
+			use_unicode=True, charset='utf8mb4', read_default_file="/etc/mysql/my.cnf")
 		self._conn.converter[246]=float
```

Comment:
I guess that should do it. I'm just not sure how it would behave without the parameter or with an empty value (i.e., would `my_cnf_path` be "None" or an empty string? And how would that affect the `MySQLdb.connect` function?)

I'm new to Frappe / ERPNext, so I'm still reading the docs and source code to get acclimated.